openlibrary/templates/support.html

```diff
@@ -14,15 +14,21 @@ <h1>$_("How can we help?")</h1>
  <form action="" method="post" name="spamForm" class="olform validate">
 <p>$:_('Please check our <a href="/help/">Help Pages</a> and <a href="/help/faq">Frequently Asked Questions</a> (FAQ) to see if your question is answered there. Thank you.')</p>
 
+ <div class="formElement">
+ <div class="label"><label for="name">$_("Your Name")</label></div>
+ <div class="input"><input type="text" class="input" name="name" id="name"/> ($_('Please fill this field if you are a patron'))</div>
```

Comment:
Don't need a description for this one.
```suggestion
 <div class="input"><input type="text" class="input" name="name" id="name"/></div>
```